Exercise_C.js

```diff
@@ -13,6 +13,21 @@ If the number is divisible by 3 and 5 it should log Hong Kong.
 See README.md if you need help running your code.
 */
 
-for (let i = 1; i <= 30; i++) {
   // CODE HERE
-}
+var output = []
+
+  for (let i = 1; i <= 30; i++) {
+    if (i%3 === 0 && i%5 ===0){
+      output.push(i, "Hong Kong");
+    }   
+    
+    else if (i%3 === 0){
+      output.push(i, "Hong");
+    }
+
+    else if (i%5 === 0){
+      output.push(i, "Kong");
+    }
+  }
+
+    console.log(output)
```

Comment:
Fantastic logic your answer is providing the correct answer. Will done for constructing an array. I would alter lines 21, 25 and 29, at the moment you are pushing in two elements, something to make your output a little cleaner and shorter would be to push in an object {i: i, value: 'Hong Kong' }